src/components/TrackPain/NewPainEntry.js

```diff
@@ -544,7 +544,7 @@ class NewPainEntry extends Component {
                 painIsInData={this.state.painIsIn}
               />
               <Paper className={classes.paperPadding} elevation={1}>
-                <Typography variant="body2" component="h3">
+                <Typography variant="body1">
                   Parts of your body affected:
                 </Typography>
                 <Typography component="p">{keysInPainIsIn}</Typography>
```

Comment:
I think this also must be changed, we must assign a v2 variant and eliminate the component attribute. This line will change in the next PR I will do, but It could be good to always specify a valid variant in the future.

Reply:
I didn't see that one! :( I'll leave it to you for the next PR